sass/sass_repositories.bzl

```diff
@@ -14,12 +14,15 @@
 
 "Install Sass toolchain dependencies"
 
-load("@build_bazel_rules_nodejs//:defs.bzl", "yarn_install")
+load("@build_bazel_rules_nodejs//:defs.bzl", "yarn_install", "check_rules_nodejs_version")
 
 def sass_repositories():
     """Set up environment for Sass compiler.
     """
 
+    # 0.31.1: entry_point attribute of rules_nodejs is now a label
+    check_rules_nodejs_version("0.31.1")
```

Comment:
If a rules_sass user is explicitly installing `build_bazel_rules_nodejs` in their WORKSPACE then they'll need to update to at least version 0.31.1. If they are not explicitly installing it then `sass_repositories()` will install the correct version transitively.